src/graph/executor/Executor.cpp

```diff
@@ -682,6 +682,27 @@ void Executor::dropBody(const PlanNode *body) {
   }
 }
 
+bool Executor::movable(const Variable *var) {
+  // Only support input variables of current executor
```

Comment:
what if the data flow and logical flow are different, such as bfsShortestPath operator

Reply:
We use reference count to identify it.